scripts/upgrade.sh

```diff
@@ -127,12 +127,13 @@ function main() {
     configure_no_proxy
     ${K8S_DISTRO}_addon_for_each addon_fetch
     host_preflights "${MASTER:-0}" "1" "1"
-    install_cri
+    install_host_dependencies
```

Comment:
im making the order here consistent with install. setup_kubeadm_kustomize must come before addon_join which must come before maybe upgrade